src/stud.cpp

```diff
@@ -0,0 +1,25 @@
+ #include "stud.hpp"
+ #include <iostream>
+ 
+ Student::Student()
+         :Student {"name","",""}{
+         }
+ Student::Student(std::string nameVal, std::string surnameVal, std::string       discordNameVal)
+         :name{nameVal},    surname{surnameVal},    discordName{discordNameVal}
+         {
+         }
+ Student::Student(const Student &source)
+       :name{source.name},    surname{source.surname},    discordName{source.    discordName}
```

Comment:
you should prefer to use = default for copy operations. They will do exactly the same thing will less code.